rflx/pyrflx/pyrflx.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+from typing import Dict, List
+
+from rflx.parser import Parser
+
+from .message import Message
+from .package import Package
+
+
+class PyRFLX:
```

Comment:
It is quite cumbersome to import this class: `from rflx.pyrflx.pyrflx import PyRFLX`. This could be improved by adding `from .pyrflx import PyRFLX` to `__init__.py`, so that only `from rflx.pyrflx import PyRFLX` is needed.

Reply:
Fixed by https://github.com/Componolit/RecordFlux/pull/158/commits/91889be32908f1f27383b00a59a97eca8d085b77.